e2e/jest.test-setup.ts

```diff
@@ -17,8 +17,7 @@ const userAgent =
  * - waitFor functions timeout
  */
 beforeEach(async () => {
-  await jestPuppeteer.resetPage();
-  await jestPuppeteer.resetBrowser();
```

Comment:
Remove `resetPage` because it causes page to close and open a new one. It's very annoying.

Reply:
Agreed, nice change.